web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -120,8 +120,8 @@ export class ShardComponent implements OnInit, OnDestroy {
     if (!isMaster) {
       // Add replication-related methods to the 'Change' section for slaves.
       result[1].items.push(
-        {label: 'Start Slave', command: (event) => {this.openStartSlaveDialog(); }},
-        {label: 'Stop Slave', command: (event) => {this.openStopSlaveDialog(); }},
+        {label: 'Start Replica', command: (event) => {this.openStartSlaveDialog(); }},
```

Comment:
```suggestion
        {label: 'Start Replication', command: (event) => {this.openStartSlaveDialog(); }},
```

Reply:
Surprisingly, GitHub didn't auto-resolve these suggestions. :) I applied them manually so I didn't add them as individual commits. I'll go ahead and resolve them as I'm pretty sure I got 'em all!